packages/shared/src/schemas/websockets.ts

```diff
@@ -24,6 +24,12 @@ export const CellUpdatePayloadSchema = z.object({
   updates: CellUpdateAttrsSchema,
 });
 
+export const CellAiGeneratePayloadSchema = z.object({
```

Comment:
Nit: Can we either drop the `cell` in the name here or change these to be `AIGenerateCell...`? This makes it clear they belong to the AI functionality rather than the normal cell operations.

Reply:
Will switch the ordering. I'd love it to be shorter and remove `cell`, but there is an `AiGenerateSrcbook` that makes it ambiguous otherwise.

Right now the `AiGenerateSrcbook` is an http endpoint, but that could change.